.gitignore

```diff
@@ -14,6 +14,7 @@
 /test/no_std/bazel-*
 /docs/bazel-*
 user.bazelrc
+MODULE.bazel.lock
```

Comment:
I believe it's recommended to commit MODULE.bazel.lock, similar to how you'd commit Cargo.lock.

Reply:
My bad. I mistakenly thought this repo was not checking them in. I see now that bzlmod-specific repos are. I will add the lock file for the new workspace.